--- calico.j2	2021-05-09 11:47:50.092447401 -0400
+++ calico.j2	2021-05-09 13:39:53.175996953 -0400
@@ -15,7 +15,7 @@
   # Configure the MTU to use for workload interfaces and tunnels.
   # By default, MTU is auto-detected, and explicitly setting this field should not be required.
   # You can override auto-detection by providing a non-zero value.
-  veth_mtu: "0"
+  veth_mtu: "{{ calico_mtu }}"
 
   # The CNI network configuration to install on each node. The special
   # values in this config will be automatically populated.
@@ -3657,7 +3657,7 @@
               value: "autodetect"
             # Enable IPIP
             - name: CALICO_IPV4POOL_IPIP
-              value: "Always"
+              value: "{{ calico_enable_ipip }}"
             # Enable or Disable VXLAN on the default IP pool.
             - name: CALICO_IPV4POOL_VXLAN
               value: "Never"
@@ -3682,8 +3682,8 @@
             # The default IPv4 pool to create on startup if none exists. Pod IPs will be
             # chosen from this range. Changing this value after installation will have
             # no effect. This should fall within `--cluster-cidr`.
-            # - name: CALICO_IPV4POOL_CIDR
-            #   value: "192.168.0.0/16"
+            - name: CALICO_IPV4POOL_CIDR
+              value: "{{ k8s_pods_subnet }}"
             # Disable file logging so `kubectl logs` works.
             - name: CALICO_DISABLE_FILE_LOGGING
               value: "true"
@@ -3695,6 +3695,9 @@
               value: "false"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            # Additional configuration for CentOS 8 using Netfilter
+            - name: FELIX_IPTABLESBACKEND
+              value: "{{ calico_iptables_backend }}"
           securityContext:
             privileged: true
           resources:
